SpotifyAPI/Local/SpotifyLocalAPI.cs

```diff
@@ -139,11 +140,18 @@ public Boolean Connect()
             return _rh.Init();
         }
 
-        public StatusResponse GetStatus()
+        /// <summary>
+        /// Gets the Spotify status
+        /// </summary>
+        /// <returns></returns>
+        public async Task<StatusResponse> GetStatus()
```

Comment:
This would break a lot of scripts and would require a new major version. What do you think about a seperate async method? Like `GetStatusAsync()`